web/template/certificate_provider_your_details.gohtml

```diff
@@ -1,27 +1,19 @@
 {{ template "page" . }}
 
 {{ define "serviceName" }}{{ tr .App "beACertificateProvider" }}{{ end }}
-{{ define "pageTitle" }}{{ tr .App "yourDetails" }}{{ end }}
+{{ define "pageTitle" }}{{ tr .App "enterYourDetails" }}{{ end }}
 
 {{ define "main" }}
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-two-thirds">
       <form novalidate method="post">
-        <h1 class="govuk-heading-xl">{{ tr .App "yourDetails" }}</h1>
-
-        <h2 class="govuk-heading-m">{{ tr .App "donor" }}</h2>
-        <p class="govuk-body">{{ .Lpa.Donor.FullName }}</p>
-
-        <h2 class="govuk-heading-m">{{ tr .App "yourDetails" }}</h2>
-        <p class="govuk-body">{{ .Lpa.CertificateProvider.FullName }}</p>
+        <h1 class="govuk-heading-xl">{{ tr .App "enterYourDetails" }}</h1>
 
         {{ template "date" (input . "date-of-birth" "dateOfBirth" .Form.Dob "hint" "dateOfBirthHint") }}
 
-        {{ template "dob-warning" . }}
-
-        {{ template "input" (input . "email" "email" .Form.Email "classes" "govuk-input--width-20" "type" "email" "spellcheck" "false" "autocomplete" "email") }}
+        {{ template "warning" (warning .App "18OrOverWarning") }}
```

Comment:
I know this is the design that they've given, but I hate that there is a warning _and_ an error. I said toss a coin and pick one option!

Reply:
Shall we see what users think next week 😉